frontend/package.json

```diff
@@ -5,7 +5,12 @@
   "dependencies": {
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.2.0"
+    "react-redux": "^7.1.3",
```

Comment:
I'd recommend to install only libraries you need on an *immediate* basis, but that's just a piece of advice